MachObfuscator/Controller/Obfuscator.swift

```diff
@@ -30,6 +30,17 @@ class Obfuscator {
         LOGGER.info("\(symbols.blacklist.classes.count) unobfuscable classes")
 
         LOGGER.info("Mangling symbols...")
+        symbols.whitelist.selectors.filter {
+            $0.utf8.count != $0.count
```

Comment:
I don't think leaving those changes here is a good idea. I imagine they were helpful during development, but they add little value on production. Additionally those computations are going to be executed every time (even when log level is non-debug). Would you mind removing those changes? I am going to add obfuscation reports, so I think this will become a little redundant.

Alternatively, if you really think those changes can be helpful in the future, would you mind improving them a little bit by applying following changes?:
1) checking for a debug log level before executing those changes?
2) extracting those changes to `ObfuscationSymbols` extension?
3) extracting `$0.utf8.count != $0.count` as `String.containsAsciiCharacters`?

Reply:
These logs were added out of curiosity. I agree, that a comprehensive reporting will be much better for obtaining such data. I will remove the logs.